tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -144,7 +144,11 @@ def test_streams_runs_with_broker_down_initially(self):
     def test_streams_should_scale_in_while_brokers_down(self):
         self.kafka.start()
 
-        configs = self.get_configs(extra_configs=",application.id=shutdown_with_broker_down")
+        # TODO KIP-441: consider rewriting the test for HighAvailabilityTaskAssignor
+        configs = self.get_configs(
+            extra_configs=",application.id=shutdown_with_broker_down" +
+                          ",internal.task.assignor.class=org.apache.kafka.streams.processor.internals.assignment.StickyTaskAssignor"
+        )
```

Comment:
This one already had a different mechanism to add more configs, so I just left it alone.